__TESTS__/unit/actions/Color.test.ts

```diff
@@ -0,0 +1,64 @@
+import TransformableImage from '../../../src/transformation/TransformableImage';
+import CloudinaryConfig from "../../../src/config/CloudinaryConfig";
+import {Color} from "../../../src/actions/Actions";
+import Colors from '../../../src/actions/color/Colors';
```

Comment:
This will not work for our users,

`import Colors from '@cloudinary/base/actions/color/Colors'` ill fail as it's not a real path.

Perhaps, to avoid confusion, /Actions/ should only contain Actions?
We could have a separate place for Constants
Aside from fixing it now, do you have any idea on how we can make sure this doesn't happen again?

Reply:
The only thing I can think of is having a consumer as part of the workspace.. but it will require upkeeping two separate test cases. Also, we can test/search for a tool such as this- https://www.npmjs.com/package/pack-n-play this will enable us to verify that our library can be used by dependent code (the d.ts we're feeding it but this might not be enough)